packages/internals/src/get-generators/utils/fixBinaryTargets.ts

```diff
@@ -11,7 +11,7 @@ export function fixBinaryTargets(
 ): BinaryTargetsEnvValue[] {
   binaryTargets = binaryTargets || []
 
-  if (!binaryTargets.find((object) => object.value === 'native')) {
+  if (!binaryTargets.find((object) => object.native === true)) {
```

Comment:
Nit, similar to above:

```suggestion
  if (!binaryTargets.find((object) => object.native)) {
```